<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GGMFlowPartModeller</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_8">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="tabPosition">
       <enum>QTabWidget::North</enum>
      </property>
      <property name="tabShape">
       <enum>QTabWidget::Rounded</enum>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="impellerTab">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
       <attribute name="title">
        <string>Меридианная проекция</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <property name="topMargin">
         <number>14</number>
        </property>
        <property name="rightMargin">
         <number>12</number>
        </property>
        <property name="bottomMargin">
         <number>13</number>
        </property>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QGroupBox" name="groupBox_2">
            <property name="maximumSize">
             <size>
              <width>300</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="title">
             <string>Размеры</string>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_7">
             <item>
              <widget class="QTreeView" name="impellerDimTreeView"/>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="groupBox">
            <property name="maximumSize">
             <size>
              <width>300</width>
              <height>150</height>
             </size>
            </property>
            <property name="title">
             <string>Параметры расчета </string>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_10">
             <item>
              <widget class="QTreeView" name="impellerCalcSettingsTreeView"/>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="evaluateImpellerButton">
            <property name="text">
             <string>Построить</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_3">
          <property name="title">
           <string>Графика</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <widget class="QTabWidget" name="tabWidget_2">
               <property name="tabPosition">
                <enum>QTabWidget::South</enum>
               </property>
               <property name="currentIndex">
                <number>2</number>
               </property>
               <property name="elideMode">
                <enum>Qt::ElideNone</enum>
               </property>
               <property name="usesScrollButtons">
                <bool>true</bool>
               </property>
               <property name="documentMode">
                <bool>false</bool>
               </property>
               <property name="tabsClosable">
                <bool>false</bool>
               </property>
               <property name="movable">
                <bool>false</bool>
               </property>
               <property name="tabBarAutoHide">
                <bool>false</bool>
               </property>
               <widget class="QWidget" name="meridianLinesPlotTab">
                <attribute name="title">
                 <string>Меридианная прекция</string>
                </attribute>
                <layout class="QHBoxLayout" name="horizontalLayout_3">
                 <item>
                  <widget class="QCustomPlot" name="meridianLinesPlot" native="true"/>
                 </item>
                </layout>
               </widget>
               <widget class="QWidget" name="meridianAreaPlotTab">
                <attribute name="title">
                 <string>График площади</string>
                </attribute>
                <layout class="QHBoxLayout" name="horizontalLayout_6">
                 <item>
                  <widget class="QCustomPlot" name="meridianAreaPlot" native="true"/>
                 </item>
                </layout>
               </widget>
               <widget class="QWidget" name="meridianLinesSpeedPlotTab">
                <attribute name="title">
                 <string>График скорости</string>
                </attribute>
                <layout class="QHBoxLayout" name="horizontalLayout_9">
                 <item>
                  <widget class="QCustomPlot" name="meridianLinesSpeedPlot" native="true"/>
                 </item>
                </layout>
               </widget>
               <widget class="QWidget" name="impeller3DModelPlotTab">
                <attribute name="title">
                 <string>3D модель</string>
                </attribute>
                <layout class="QHBoxLayout" name="horizontalLayout_5">
                 <property name="leftMargin">
                  <number>6</number>
                 </property>
                 <property name="topMargin">
                  <number>6</number>
                 </property>
                 <property name="rightMargin">
                  <number>6</number>
                 </property>
                 <property name="bottomMargin">
                  <number>6</number>
                 </property>
                 <item>
                  <widget class="QOpenGLWidget" name="openGLWidget"/>
                 </item>
                </layout>
               </widget>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="statorTab">
       <attribute name="title">
        <string>Спиральный отвод</string>
       </attribute>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Справка</string>
    </property>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
